docs/download.md

```diff
@@ -2,15 +2,23 @@
 
 ## Current development
```

Comment:
I think we can change the other of the download items : before code release and then cur dev binaries